internal/servers/private_bare_metal_instance_types_server.go

```
@@ -0,0 +1,413 @@
+				"field 'spec.hardware.memory' is immutable and cannot be cleared")
+		}
+		if mergedMem.GetTotalGb() != existingMem.GetTotalGb() {
+			return grpcstatus.Errorf(grpccodes.InvalidArgument,
+				"field 'spec.hardware.memory.total_gb' is immutable and cannot be changed from '%d' to '%d'",
+				existingMem.GetTotalGb(), mergedMem.GetTotalGb())
+		}
+	}
+
+	return nil
+
+	return nil
```

golangci-lint (2.12.2)

[error] unreachable: unreachable code

(govet)